config.json

```diff
@@ -11,6 +11,7 @@
     "run-length-encoding",
     "list-ops",
     "sublist",
+    "flatten-array",
```

Comment:
I think maybe putting this before sublist makes slightly more sense, it'd be a good warmup for sublist

thoughts?